diff --git a/Makefile b/Makefile
index fd59628..2512c88 100644
--- a/Makefile
+++ b/Makefile
@@ -19,6 +19,7 @@ AVPROGS-$(CONFIG_FFSERVER) += ffserver
 
 AVPROGS    := $(AVPROGS-yes:%=%$(PROGSSUF)$(EXESUF))
 INSTPROGS   = $(AVPROGS-yes:%=%$(PROGSSUF)$(EXESUF))
+AVPROGS    += lufscalc$(PROGSSUF)$(EXESUF)
 PROGS      += $(AVPROGS)
 
 AVBASENAMES  = ffmpeg ffplay ffprobe ffserver
@@ -118,6 +119,9 @@ $(PROGS): %$(PROGSSUF)$(EXESUF): %$(PROGSSUF)_g$(EXESUF)
 	$(CP) $< $@
 	$(STRIP) $@
 
+lufscalc.o: CFLAGS += -Ibs1770/ -DFFMPEG_STATIC_BUILD
+lufscalc_g$(EXESUF): LDFLAGS += bs1770/*.o
+
 %$(PROGSSUF)_g$(EXESUF): %.o $(FF_DEP_LIBS)
 	$(LD) $(LDFLAGS) $(LDEXEFLAGS) $(LD_O) $(OBJS-$*) $(FF_EXTRALIBS)
 
