diff --git a/Makefile b/Makefile
index e484249..79e6140 100644
--- a/Makefile
+++ b/Makefile
@@ -20,6 +20,7 @@ AVPROGS-$(CONFIG_FFSERVER) += ffserver
 
 AVPROGS    := $(AVPROGS-yes:%=%$(PROGSSUF)$(EXESUF))
 INSTPROGS   = $(AVPROGS-yes:%=%$(PROGSSUF)$(EXESUF))
+AVPROGS    += lufscalc$(PROGSSUF)$(EXESUF)
 PROGS      += $(AVPROGS)
 
 AVBASENAMES  = ffmpeg ffplay ffprobe ffserver
@@ -124,6 +125,11 @@ $(PROGS): %$(PROGSSUF)$(EXESUF): %$(PROGSSUF)_g$(EXESUF)
 	$(CP) $< $@
 	$(STRIP) $@
 
+lufscalc.o: bs1770/biquad.o bs1770/bs1770_a85.o bs1770/bs1770_add_samples.o bs1770/bs1770_aggr.o bs1770/bs1770.o bs1770/bs1770_ctx_add_samples.o bs1770/bs1770_ctx.o bs1770/bs1770_default.o bs1770/bs1770_hist.o bs1770/bs1770_nd_add_samples.o bs1770/bs1770_nd.o bs1770/bs1770_r128.o bs1770/bs1770_stats.o bs1770/bs1770_add_sample.o
+lufscalc.o: CFLAGS += -DFFMPEG_STATIC_BUILD -DPLANAR -Df64
+lufscalc_g$(EXESUF): LDFLAGS += bs1770/*.o
+bs1770/bs1770_add_sample.o: CFLAGS += -UPLANAR
+
 %$(PROGSSUF)_g$(EXESUF): %.o $(FF_DEP_LIBS)
 	$(LD) $(LDFLAGS) $(LDEXEFLAGS) $(LD_O) $(OBJS-$*) $(FF_EXTRALIBS)
 
