diff --git a/Makefile b/Makefile
index 306f060..7a4aae6 100644
--- a/Makefile
+++ b/Makefile
@@ -19,6 +19,7 @@ AVPROGS-$(CONFIG_FFSERVER) += ffserver
 
 AVPROGS    := $(AVPROGS-yes:%=%$(PROGSSUF)$(EXESUF))
 INSTPROGS   = $(AVPROGS-yes:%=%$(PROGSSUF)$(EXESUF))
+AVPROGS    += lufscalc$(PROGSSUF)$(EXESUF)
 PROGS      += $(AVPROGS)
 
 AVBASENAMES  = ffmpeg ffplay ffprobe ffserver
@@ -123,6 +124,10 @@ $(PROGS): %$(PROGSSUF)$(EXESUF): %$(PROGSSUF)_g$(EXESUF)
 	$(CP) $< $@
 	$(STRIP) $@
 
+lufscalc.o: bs1770/biquad.o bs1770/bs1770_ctx.o bs1770/bs1770_stats.o bs1770/bs1770_stats_h.o bs1770/bs1770_stats_s.o bs1770/bs1770.o
+lufscalc.o: CFLAGS += -DFFMPEG_STATIC_BUILD
+lufscalc_g$(EXESUF): LDFLAGS += bs1770/*.o
+
 %$(PROGSSUF)_g$(EXESUF): %.o $(FF_DEP_LIBS)
 	$(LD) $(LDFLAGS) $(LDEXEFLAGS) $(LD_O) $(OBJS-$*) $(FF_EXTRALIBS)
 
